pymc/tests/test_distributions_moments.py

```diff
@@ -612,3 +613,22 @@ def test_discrete_uniform_moment(lower, upper, size, expected):
     with Model() as model:
         DiscreteUniform("x", lower=lower, upper=upper, size=size)
     assert_moment_is_expected(model, expected)
+
+
+@pytest.mark.parametrize(
+    "p, size, expected",
+    [
+        (np.arange(0.1, 0.4, 0.1), None, 3),
```

Comment:
Looks good but can we perhaps have some of the tests have the moment not be the last element?

Also for readability defining the p explicitly such as `np.array([.6, .1, .3])` seems better.

Reply:
Ah sure, I agree it is not really readable.